django/http/response.py

```diff
@@ -628,13 +632,20 @@ def set_headers(self, filelike):
 class HttpResponseRedirectBase(HttpResponse):
     allowed_schemes = ["http", "https", "ftp"]
 
-    def __init__(self, redirect_to, preserve_request=False, *args, **kwargs):
+    def __init__(
+        self,
+        redirect_to,
+        preserve_request=False,
+        *args,
+        max_length=MAX_URL_REDIRECT_LENGTH,
```

Comment:
Any interest in passing through `kwargs` from `redirect()` as part of this change (following the example of `preserve_request` in ticket-35784?